scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/nsop.yaml

```diff
@@ -0,0 +1,28 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'NSOP'
+
+NSOP-16_3.9x9.9mm_P1.27mm:
+  size_source: 'JEDEC MS-012AC, https://www.holtek.com/documents/10179/11842/HT42B534-xv110.pdf'
+  body_size_x: # from JEDEC
+    nominal: 3.9
+    tolerance: 0.1
+  body_size_y:
+    minimum: 9.8
+    maximum: 10
+  overall_height:
+    maximum: 1.75
+
+  overall_size_x: # from JEDEC (agrees with linked datasheet)
+    nominal: 6
+    tolerance: 0.2
+  lead_len:
```

Comment:
This is also equivalent to JEDEC. `0.835+/-0.835`. Please add a comment here as well.